src/dut.rs

```diff
@@ -180,10 +181,10 @@ const CMD_GET_DEFAULT_IFACE: &str =
     r"ip route get 8.8.8.8 | sed -E 's/^.* dev ([^ ]+) .*$/\1/' | head -n 1";
 
 // Only keys that are always available can be listed here
-const DEFAULT_DUT_INFO_KEYS: [&str; 8] = [
+// We should consider the value of these keys if DUT is a VM
```

Comment:
This comment is not needed because this is not only for the VM.

Reply:
OK, I removed it!